flattentool/lib.py

```diff
@@ -17,4 +17,6 @@ def parse_sheet_configuration(configuration_list):
             configuration['ignore'] = True
         if (len(parts) == 1 and parts[0].lower() in ("hashcomments", "hashcomment")):
             configuration['hashcomments'] = True
+        if (len(parts) == 2 and parts[0].lower() == "xmlrootpath"):
+            configuration['XMLRootPath'] = parts[1]
```

Comment:
Could you also add one of these for RootListPath also?

Reply:
Sure!

Reply:
Fixed in 795644f and 769b272.